authors/apps/articles/migrations/0005_auto_20190423_0847.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.1.7 on 2019-04-23 05:47
```

Comment:
Please delete both files and generate one file.
To do this - `python manage.py migrate articles 005` and  `python manage.py migrate articles 004`
- If you see that logs have successfully reverted, delete these two migration files and then makemigrations to generate a new file.